### PR TITLE
FED-1855 Update instructions for dart2_upgrade codemod

### PR DESCRIPTION
**Pull request by greglittlefield-wf**
## Motivation
The README referenced a codemod that had been removed, and didn't describe a path forward.

## Changes
Update README to show how to run that codemod.

## Review
_[See CONTRIBUTING.md][contributing-review-types] for more details on review types (+1 / QA +1 / +10) and code review process._

  

Please review: 

### QA Checklist
- [ ] Tests were updated and provide good coverage of the changeset and other affected code
- [ ] Manual testing was performed if needed
    - [ ] Steps from PR author: 
        
    - [ ] Anything falling under manual testing criteria [outlined in CONTRIBUTING.md][contributing-manual-testing]

## Merge Checklist
While we perform many automated checks before auto-merging, some manual checks are needed:
- [ ] A Frontend Frameworks Design member has reviewed these changes
- [ ] There are no unaddressed comments _- this check can be automated if reviewers use the "Request Changes" feature_
- [ ] _For release PRs -_ Version metadata in Rosie comment is correct

[contributing-review-types]: https://github.com/Workiva/over_react_codemod/blob/master/CONTRIBUTING.md#review-types
[contributing-manual-testing]: https://github.com/Workiva/over_react_codemod/blob/master/CONTRIBUTING.md#manual-testing-criteria

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: README.md [hunk 1]
```diff
@@ -29,31 +29,13 @@ For more information, please see the [INTL Message Codemod Readme](docs/intl_mes
...
-    flag once you're ready to drop Dart 1 support.
-
-For more information on the transition from Dart 1 to Dart 2 and how it affects
-over_react, check out the [over_react Dart 2 migration guide][over_react_dart2].
-It includes sample diffs of the changes that these codemods will introduce.
+To use it, activate over_react_codemod 1.12.1 instead of the latest:
+```sh
+dart pub global activate over_react_codemod ^1.12.1
+```
+and see its documentation: https://github.com/Workiva/over_react_codemod/tree/1.12.1#dart-1-to-dart-2-codemod
```
Comment (kealjones-wk) [inline README.md:38]: After reading the thread with pat is this accurate? or was the final solution to use 1.1.0? also do they need to use a specific dart version? (2.1.0 Was called out in the chat i think?)

**Review by kealjones-wk — CHANGES_REQUESTED** [review]
Some stuff after the chat continued

File: README.md (hunk 1 above)
Comment (greglittlefield-wf) [reply README.md:38]: Mm good catch, will update

Comment (greglittlefield-wf) [reply README.md]: Updated; https://github.com/Workiva/over_react_codemod/pull/263/commits/84685b9302e65050f562a4a10d1fc74c9e9c0804

**Review by kealjones-wk — APPROVED** [review]
+10

**Comment by greglittlefield-wf** [comment]
@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**